commands/create.go

```diff
@@ -142,6 +142,12 @@ func runCreate(dockerCli command.Cli, in createOptions, args []string) error {
 				return err
 			}
 		}
+
+		if in.driver == "kubernetes" {
+			// naming endpoint to make --append works
+			ep = fmt.Sprintf("%s://%s/%s", in.driver, in.name, in.nodeName)
```

Comment:
%s://%s?nodeselector=%s

Reply:
sure, just a unique string here.

nodeselector not required.

how about `%s://deployments/%s` ?

Reply:
%s://%s?deployment=%s